pyproject.toml

```diff
@@ -96,3 +96,7 @@ markers = [
   "slow: marks tests as slow (deselect with '-m \"not slow\"')",
   "numpy: marks tests that require numpy",
 ]
+
+[tool.pyright]
+include = ["src/**"]
+exclude = ["tests/**"]
```

Comment:
suggestion(non-blocking): we should probably include tests to make catch any incorrect usages